### DISCUSSION TITLE
Is there a way to reduce the sidebar-splitter2 width below minimum? Its huge!

### DISCUSSION THREAD

**Discussion by sakanatha** [Floorp-Projects/Floorp#59] [Q&A] [answered]
I tried to lower it using "#sidebar-splitter2{width: 1px !important;}" in userchrome.css, even tried to set minimum to none in browser.xhtml but any values below what it looks like 10px are ignored, higher values works just fine. 
I'm also unable to change its color in css, but the main issue is its width, the perfect solution would be to make it transparent
![XrKFcXj64Gx1zR2e0NT8DIKml89ktjDF8iiNHviQ](https://user-images.githubusercontent.com/113897465/191439922-33190fb6-0130-4ab5-8b76-34daa306d42e.jpg)

**Comment by Ryosuke-Asano** [comment]
This element is a fixed element in XHTML, and we can't make it smaller.

We recommend that you leave only the judgment of the elements so that you can't see it

**Reply by sakanatha** [reply]
So is it possible to change its width in xhtml?
As for the second sentence, sorry, I didn't understand you

**Reply by Ryosuke-Asano** [reply]
You can't manipulate XHTML.

If you don't change the size of the sidebar, you can hide this element.

I don't think I can change the width, but I'll study it.

**Reply by sakanatha** [reply]
But just hiding it will leave me without options to dynamically resize the sidebar (by dragging it). 

So, as I understand, theres completely no way for me to change its width even if im willing to edit browser.xhtml manually?

**Reply by Ryosuke-Asano** [reply]
It's probably because this element manages drugs...

max-height: 1px !important; 

Does it work?

**Reply by sakanatha** [reply]
Yes, splitter is bound to resizing for the main sidebar as well

max-height: 1px !important; just breaks everything below
![pbA6SrbUDZiuoHmpxRCQdzD5mAgDS8SqqpsjTXgv](https://user-images.githubusercontent.com/113897465/191449789-59c19859-006e-4368-aeb5-a42b440a43c0.jpg)
max-height: 500px
![CL5yvR2jUmiDIeIWmk9N6Sh6icl7DwbHeoahgSdE](https://user-images.githubusercontent.com/113897465/191449792-7f564789-1819-46f2-acd8-7706934aefda.jpg)

**Reply by Ryosuke-Asano** [reply]
Hmm... Are you going to break it? I'm on my way home from school now, so I'll go home and check if it's possible after finishing the cram school.

I'm sorry I couldn't help you.

By the way, why is the logo Firefox?

**Reply by sakanatha** [reply]
I changed some things while experimenting with floorp, mainly trying to adapt my custom theme, some of them changed logo but it has no effect on the problem, tried with clean floorp as well. Thanks
![M4AvOJrH0GAQ1ROtYXHhll06SSgbp0rsplX9CLJm](https://user-images.githubusercontent.com/113897465/191452614-97b92214-32ce-4c1a-a7d6-9f42aa734dfc.jpg)

**Comment by sakanatha** [comment, marked as answer]
Ok, it turned out to be a very stupid question, I was sure that Ive tried these options already, at least in xhtml, but no, either i havent or they get overridden outside of xhtml.

Fixed it by setting this options in userchrome.css:
min-width: 1px !important;
max-width: 1px !important;

**Reply by Ryosuke-Asano** [reply]
Wow

Sry I typed height but.... I was thinking about width in my head...

😱
